src/snowflake/snowpark/_internal/udf_utils.py

```diff
@@ -972,23 +1131,26 @@ def resolve_imports_and_packages(
         inline_code,
         all_imports,
         all_packages,
+        udf_level_imports,
```

Comment:
Shouldn't `udf_level_imports` be included in `all_imports`?

Reply:
Added inline explanation for this.